### PR TITLE
Push notifications to Android via GCM

### PR DESCRIPTION
**Pull request by joaojusto**

### CODE REVIEW

**Comment by joaojusto** [comment]
@gabrielpoca

File: client/lib/push_plugin/push_notification.coffee [hunk 1]
```diff
@@ -0,0 +1,50 @@
+@onNotification = (e) ->
```
Comment (gabrielpoca) [inline client/lib/push_plugin/push_notification.coffee]: why are there two files with the same name on the same folder (one with snake case the other with camel case)? Also, don't use camel case on files or folder's names.

File: collections/users.coffee [hunk 2]
```diff
@@ -10,6 +16,15 @@ Meteor.methods
 
     insertIntoTrip(user.id, tripId)
 
+  insertPushNotificationsKey: (regId) ->
+    console.log regId
```
Comment (gabrielpoca) [inline collections/users.coffee]: Don't forget to remove this unnecessary logs.

File: client/views/users/sign_in.coffee [hunk 3]
```diff
@@ -20,9 +20,24 @@ Template.signIn.submit = ->
     if err
       console.log(err)
     else
+      Template.signIn.pushNotificationRegister()
       Router.go 'tripsIndex'
   )
 
+Template.signIn.pushNotificationRegister = ->
+  try
```
Comment (gabrielpoca) [inline client/views/users/sign_in.coffee]: Why is the `try` necessary? I also think this logic does not belong in the template. This is not view related.

File: client/views/lab/lab.coffee [hunk 4]
```diff
@@ -18,3 +18,6 @@ Template.lab.helpers
 
     categories: ->
       Categories.all()
+
+    onNotification: ->
```
Comment (gabrielpoca) [inline client/views/lab/lab.coffee]: Why do you include this on the lab?

**Comment by joaojusto** [comment]
@gabrielpoca

**Comment by joaojusto** [comment]
@gabrielpoca

**Comment by joaojusto** [comment]
@gabrielpoca

**Comment by gabrielpoca** [comment]
:+1:

**Pull request closed without merging**